auth/spicedb/policies.go

```diff
@@ -132,24 +132,13 @@ func (pa *policyAgent) AddPolicy(ctx context.Context, pr auth.PolicyReq) error {
 }
 
 func (pa *policyAgent) DeletePolicies(ctx context.Context, prs []auth.PolicyReq) error {
-	updates := []*v1.RelationshipUpdate{}
 	for _, pr := range prs {
-		updates = append(updates, &v1.RelationshipUpdate{
-			Operation: v1.RelationshipUpdate_OPERATION_DELETE,
-			Relationship: &v1.Relationship{
-				Resource: &v1.ObjectReference{ObjectType: pr.ObjectType, ObjectId: pr.Object},
-				Relation: pr.Relation,
-				Subject:  &v1.SubjectReference{Object: &v1.ObjectReference{ObjectType: pr.SubjectType, ObjectId: pr.Subject}, OptionalRelation: pr.SubjectRelation},
-			},
-		})
-	}
-	if len(updates) == 0 {
-		return errors.Wrap(errors.ErrMalformedEntity, errNoPolicies)
-	}
-	_, err := pa.permissionClient.WriteRelationships(ctx, &v1.WriteRelationshipsRequest{Updates: updates})
-	if err != nil {
-		return errors.Wrap(errRemovePolicies, handleSpicedbError(err))
+		err := pa.DeletePolicy(ctx, pr)
+		if err != nil {
+			return err
+		}
 	}
+
```

Comment:
@WashingtonKK 
`DeletePolicies` is different from `DeletePolicy`
`DeletePolicies` supports filteing of options, But DeletePolicy doesn't support filtering options